src/TestStack.BDDfy/Processors/UnusedExampleException.cs

```diff
@@ -1,21 +1,12 @@
 ﻿namespace TestStack.BDDfy.Processors
 {
     using System;
-    using System.Runtime.Serialization;
 
-    [Serializable]
```

Comment:
Create an attribute in your code in the right namespace. Does nothing but don't need ifs.